tine20/Addressbook/Convert/Contact/VCard/Abstract.php

```diff
@@ -176,10 +176,73 @@ public function toTine20Model($blob, Tinebase_Record_Abstract $_record = null, $
+                        Tinebase_Core::getLogger()->debug(__METHOD__ . '::' . __LINE__ . ' Photo: ' . $property['Endocing'] . ":" . $property['TYPE'] );
+                    if (    ( $property['ENCODING'] != "b" )
+                         && ( $property['ENCODING'] != "B" ) ) {
+                        break;
+                    }
+                    switch ( $property['TYPE'] ) {
+                        case 'JPG' : {}
+                        case 'jpg' : {}
+                        case 'Jpg' : {}
+                        case 'Jpeg' : {}
+                        case 'jpeg' : {}
+			case 'PNG' : {}
+			case 'png' : {}
+                        case 'JPEG' : {
+                            if (Tinebase_Core::isLogLevel(Zend_Log::DEBUG)) 
+                                Tinebase_Core::getLogger()->debug(__METHOD__ . '::' . __LINE__ . ' Photo: passing on jpeg image as is (' . strlen($property->getValue()) .')' );
+                            $jpegphoto = $property->getValue();
+                            break;
+                        }
+                        default : {
+                            if (Tinebase_Core::isLogLevel(Zend_Log::DEBUG)) 
+                                Tinebase_Core::getLogger()->debug(__METHOD__ . '::' . __LINE__ . ' Photo: recoding to jpeg (' . strlen($property->getValue()) . ')' );
+/*
+			    $binimage = imagecreatefromstring($property->getValue());
+                            if ( $binimage === false ) {
+                                $jpegphoto = null;
+                            if (Tinebase_Core::isLogLevel(Zend_Log::DEBUG)) 
+                                Tinebase_Core::getLogger()->debug(__METHOD__ . '::' . __LINE__ . ' Photo: imageformat unkown or not supported ' );
+                                break;
+                            }*/
+
+                            $info = getimagesizefromstring($property->getValue());
+                            if (Tinebase_Core::isLogLevel(Zend_Log::DEBUG)) 
+                                Tinebase_Core::getLogger()->debug(__METHOD__ . '::' . __LINE__ . ' Photo: is of type (' . $info['mime'] . ')' );
+                            if ( in_array($info['mime'], Tinebase_ImageHelper::getSupportedImageMimeTypes()) ) {
+                                if (Tinebase_Core::isLogLevel(Zend_Log::DEBUG)) 
+                                    Tinebase_Core::getLogger()->debug(__METHOD__ . '::' . __LINE__ . ' Photo: not recoded ' );
+                            	$jpegphoto = $property->getValue();
+                            	break;
+                            }
+                            $recode = imagecreatefromstring($property->getValue());
+                            if ( $recode === false ) {
+                                break;
+                            }                            
+                            ob_start();
+                            if ( imagejpeg($recode,null,90) === true ) {
```

Comment:
PSR-1, PSR-2 please!

Reply:
Just a quick question, psr-1 psr-2 what do you mean by that or are they implicitly applied if i rebase my branche to latest head or will that show by the Tarvis CI anyway?